snmp/tests/conftest.py

```diff
@@ -19,7 +19,6 @@
 )
 
 FILES = [
-    "https://ddintegrations.blob.core.windows.net/snmp/f5.snmprec",
```

Comment:
Note: we're having issues uploading the new file to Azure Storage. We're going to use the file pushed here instead for now.